docs/secrets-sops-encryption.md

```diff
@@ -0,0 +1,96 @@
+## Secrets SOPS Encryption
+
+Install sops:
+```
+brew install sops
+```
+
+#### Steps to SOPS Encrypt
+
+The following is an example for kube-prometheus-stack:
+
+In your Helm File: apps/monitoring/kube-prometheus-stack/kube-prometheus-stack.yaml
+you will define what the secret file name for kube-prometheus-stack is looking for, here is an example:
+```
+  valuesFrom:
+    - name: "prometheus-values"
+      kind: Secret
+```
+
+Create a file called values.yaml in a tmp folder
```

Comment:
Create a file called values.yaml in a temporary folder e.g. `/tmp` on macOS, Linux